src/aktualizr_secondary/socket_server.cc

```diff
@@ -0,0 +1,183 @@
+#include "socket_server.h"
+
+#include "AKIpUptaneMes.h"
+#include "asn1/asn1_message.h"
+#include "logging/logging.h"
+#include "socket_activation/socket_activation.h"
+#include "utilities/dequeue_buffer.h"
+#include "utilities/sockaddr_io.h"
+
+#include <linux/tcp.h>
+#include <sys/socket.h>
+#include <sys/types.h>
+
+void SocketServer::Run() {
+  while (true) {
+    int con_fd;
+    std::unique_ptr<sockaddr_storage> peer_sa(new sockaddr_storage);
```

Comment:
can this be allocated on the stack here?

Reply:
Good point. Done